internal/host/refresh_status_preprocessor.go

```diff
@@ -219,6 +219,14 @@ func newValidations(v *validator) []validation {
 			id:        BelongsToMajorityGroup,
 			condition: v.belongsToMajorityGroup,
 		},
+		{
+			id:        IsMtuOKAllInterfaces,
```

Comment:
Why not have a single validation that will verify according to the current cluster setting ?

Reply:
As I commented above
>  we want to separate the validations for CMN and UMN, so that we can disable one while keeping the other enabled, or make one required for installation while the other remains a warning only.

Reply:
You don't need to leave a warning for UMN if the current setting is CMN.
Having both validations seems confusing to me.
For connectivity check the belongs-to-majority-group is a single validation even though it is checked differently for CMN and UMN.

Reply:
If the validation isn't relevant to CMN/UMN, it will be skipped. See the example: https://github.com/openshift/assisted-service/pull/7012/files#diff-d7db3d1eb3c67167cb6aaffa4ed66cd1dcd7d2097d63648ecd03410a36f1283eR315-R316

Reply:
What I mean is that it was decided to separate the validations instead of combining them and separate in a condition, because we may want to make the one for CMN a requirement for installation, while for UMN, it might not be mandatory (for example). That's why they need to be separated. As I mentioned earlier, I added a condition to each validation so that if it's not relevant, it will be skipped.

Reply:
We can make it optional in the validation function by asking what type of cluster do we validate.
Is this separation a requirement ?

Reply:
> Is this separation a requirement ?
Yes
@AlonaKaplan

Reply:
As discussed, it was decided to implement a single validation for both CMN and UMN. If differentiation is needed in the future, it will be handled through a new rule.